tasks/Tasks/rhods_olm.robot

```diff
@@ -8,7 +8,7 @@ Library          String
 
 ***Variables***
 ${cluster_type}          OSD
-${operator_version}      latest
+${operator_version}      ${EMPTY}
 ${RHODS_INSTALL_REPO}    None
 *** Tasks ***
 Can Install RHODS Operator
```

Comment:
I think it should be renamed to `Install RHODS Self-Managed`

```
Install RHODS Self-Managed
  [Tags]  install-rhods-self-managed
  Given   RHODS Operator Is Not Already Installed 
  And  Selected Cluster Type ${cluster_type}
  And  Selected Operator Image Url ${image_url} 
  When Installing RHODS Operator Using Self-Managed Script   ${cluster_type}   ${image_url}
  Then RHODS Operator Should Be Installed
  [Teardown]   Install Teardown
```

Reply:
I will modify the script during refactoring. Have opened [Jira](https://issues.redhat.com/browse/RHODS-5396) against it for tracking.